src/main/java/it/pn/frontend/e2e/section/destinatario/personaGiuridica/AggiungiDelegaPGSection.java

```diff
@@ -127,6 +129,8 @@ public String waitMessaggioErroreData() {
     }
```

Comment:
inserire try catch in tutto il file dove manca

Reply:
non serve il try catch with message lo gestisce